dynamo-adapter.js

```diff
@@ -0,0 +1,131 @@
+const {promisify} = require('util');
+const {DynamoTable} = require('./slim');
+
+const _isCallback = (fn) => typeof fn === 'function';
```

Comment:
I don't think this naming scheme is useful here, we already have `exports`

Reply:
I don't mind the `_name` since it indicates a helper/private method regardless of export.

Reply:
Well I guess since they aren't exported its true that the dangling underscore is meaningless. Can change but would like to keep `_call` on the class